lib/pry/pry_class.rb

```diff
@@ -1,7 +1,19 @@
 require 'pry/config'
 class Pry
 
-  HOME_RC_FILE = ENV["PRYRC"] || "~/.pryrc"
+  HOME_RC_FILE = begin
```

Comment:
Sorry, I just realised this `begin..end` is not needed. You can simply unwrap the `if` and it'll work the same :)